library/include/internal/rocblas-beta.h

```diff
@@ -547,6 +547,146 @@ ROCBLAS_EXPORT rocblas_status
                                                   rocblas_int*      list_array,
                                                   rocblas_int*      list_size);
 
+ROCBLAS_DEPRECATED_MSG(
+    "rocblas_gemm_ex3 is a beta feature and is subject to change in future releases."
+    "Trying to run this API on unsupported hardware will return rocblas_status_arch_mismatch ")
+/*! @{
+    \brief <b> BLAS BETA API </b>
+
+    \details
+    gemm_ex3 performs one of the matrix-matrix operations
+
+        D = alpha*op( A )*op( B ) + beta*C,
+
+    where op( X ) is one of
+
+        op( X ) = X      or
+        op( X ) = X**T   or
+        op( X ) = X**H,
+
+    alpha and beta are scalars, and A, B, C, and D are matrices, with
+    op( A ) an m by k matrix, op( B ) a k by n matrix and C and D are m by n matrices.
+
+    gemm_ex3 is a temporary API to support float8 computation
+    Trying to run this API on unsupported hardware will return rocblas_status_arch_mismatch
+
+    Supported types are as follows:
```

Comment:
To me this isn't clear/exhaustive. For the last row C/D can also be fp16/fp32 right? Looking at the source code in `rocblas_gemm_ex3.hpp` I would expect something like the following, just for compute_type==fp32:
| A type | B type | C type | D type | Acc Type |
| ------ | ------ | ------ | ------ | ------ |
| fp8 | fp8 | fp32 | fp32 | fp32 |
| fp8 | fp8 | fp16 | fp16 | fp32 |
| bf8 | bf8 | fp32 | fp32 | fp32 |
| bf8 | bf8 | fp16 | fp16 | fp32 |
| f8 | bf8 | fp32 | fp32 | fp32 |
| f8 | bf8 | fp16 | fp16 | fp32 |
| bf8 | fp8 | fp32 | fp32 | fp32 |
| bf8 | fp8 | fp16 | fp16 | fp32 |
| fp8 | fp8 | fp8 | fp8 | fp32 |
| bf8 | bf8 | bf8 | bf8 | fp32 |
| fp8 | bf8 | bf8 | bf8 | fp32 |
| bf8 | f8 | bf8 | bf8 | fp32 |

But I could be overlooking something.